src/build/specialize_types/IR.zig

```diff
@@ -13,12 +13,19 @@ const Self = @This();
 env: *base.ModuleEnv,
 exposed_values: std.AutoHashMap(Ident.Idx, Expr.Idx),
 exposed_functions: std.AutoHashMap(Ident.Idx, Function),
+/// ??? all types in the module?
 types: Type.List,
+/// ??? all expressions in the module?
 exprs: Expr.List,
+/// ??? all typed expressions in the module?
 typed_exprs: Expr.Typed.List,
+/// ??? all patterns in the module?
 patterns: Pattern.List,
+/// ??? all typed patterns in the module?
 typed_patterns: Pattern.Typed.List,
+/// ??? all typed identifiers in the module?
 typed_idents: TypedIdent.List,
+/// ??? all when branches in the module?
 when_branches: WhenBranch.List,
```

Comment:
Just curious, are these required comments or comments you just want to add for clarity?

Reply:
They're not required, I just wanted to add them